ablator/modules/storage/remote.py

```diff
@@ -53,6 +105,23 @@ def _make_cmd_up(self, local_path: Path, destination: str):
         return cmd
 
     def _make_cmd_down(self, local_path: Path, destination: str, verbose=True):
+        """
+        Make the rsync command to download a file from the remote storage.
```

Comment:
The `-r` used in rsync command recurses into directories, so I think this one will download all files in the destination to the `local_path`